nextcord/file.py

```diff
@@ -63,6 +63,12 @@ class File:
         in the Discord client.
     spoiler: :class:`bool`
         Whether the attachment is a spoiler.
+    force_close: :class:`bool`
+        Whether to forcibly close the bytes used to create the file
+        when ``.close()`` is called.
+        This will also make the file bytes unusable by flusing it from
```

Comment:
```suggestion
        This will also make the file bytes unusable by flushing it from
```

Reply:
Was marked as resolved, but it doesn't seem like these were fixed yet

Reply:
Weird I committed the latest